lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -38,7 +38,8 @@ import { HttpClientWithRetries } from "../../network/HttpClientWithRetries.js";
  * Managed Identity User Assigned Id Query Parameter Names
  */
 export const ManagedIdentityUserAssignedIdQueryParameterNames = {
-    MANAGED_IDENTITY_CLIENT_ID: "client_id",
+    MANAGED_IDENTITY_CLIENT_ID_2017: "clientid",
```

Comment:
also adding a comment here to explain why we have these two might help in the future.

Reply:
done